precompiles/vesting/src/contracts/Vesting.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+/// @title Pallet Vesting Interface (pallet code here: https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/vesting)
+/// @author LAOS Team
+/// @notice This interface allows Solidity contracts to interact with pallet-vesting
+/// @custom:address 0x0000000000000000000000000000000000000407
```

Comment:
can we use `0x0000000000000000000000000000000000000406` instead? AFAIK there is no need to create a gap

Reply:
I was following the pattern of the previous two. I can change it to `...406` if you like

Reply:
the gap between `403` and `405` might be because we removed a precompile from the codebase but into the `pallet-evm` storage still existed for such precompile at that address (as far as I remember)

Reply:
fixed